patch bindgen to use BINDGEN_EXTRA_CLANG_ARGS variable to set sysroot for cross builds
https://github.com/jnbr/rust-bindgen/tree/0.43.2-clang_args

--- Cargo.toml.orig	2019-06-15 02:33:02.105715462 +0200
+++ Cargo.toml	2019-06-15 02:36:33.905701785 +0200
@@ -6,3 +6,4 @@
 "libpijul:0.12.0" = { path = "libpijul" }
 "line:0.1.15" = { path = "line" }
 "diffs:0.3.0" = { path = "diffs" }
+"bindgen:0.43.2" = {git = 'https://github.com/jnbr/rust-bindgen', rev = "1467ac97c669c1ddda8117ccf1d398609b0e6fa9"}
